keyboards/androdi/exnumpad/README.md

```diff
@@ -0,0 +1,15 @@
+# EXNUMPAD VER. 2
```

Comment:
Should follow <https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md>, and file should be lowercase.